components/canvas/utils/formatNodeText.ts

```diff
@@ -1,16 +1,8 @@
-export const formatNodeText = (
-  text: string | null,
-  maxLength = 100,
-  removeNewLines = false
-): string => {
+export const formatNodeText = (text: string | null): string => {
```

Comment:
`| null` can be removed since you're only passing strings into this function. This means you can rename the function as well.

Reply:
When I check the argument wherever this function is called the type is `string` which means it's not needed. Otherwise the TypeScript type of the argument when calling the function must be wrong.

Reply:
I have updated some API types, including the type of the variable that is passed into this function.